.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       run: |
         pip install .
         pip install -r requirements.d/dev.txt
-    - name: Test formatting with Flake8, isort and Black
+    - name: Test formatting with Flake8, isort, ruff and Black
       run: make lint
```

Comment:
I feel like pre-commit should replace the `make lint` rule.
This can also be handled in a follow up PR.